R/bootstrap.R

```diff
@@ -948,8 +951,9 @@ textOutput <- function(outputId, container = if (inline) span else div, inline =
 #'   tableOutput("view")
 #' )
 #' @export
-verbatimTextOutput <- function(outputId) {
-  textOutput(outputId, container = pre)
+verbatimTextOutput <- function(outputId, placeholder = TRUE) {
+  pre(id = outputId,
+      class = paste("shiny-text-output", if (!placeholder) "noplaceholder"))
```

Comment:
A nit: if `!placeholder`, this will result in `"shiny-text-output "`, with a trailing space. One way to work around this is to use `paste()` on a vector and use the `collapse` argument, as in:

```R
paste(c("shiny-text-output", if (!placeholder) "noplaceholder"), collapse = " ")
```